GPU/GLES/ShaderManagerGLES.cpp

```diff
@@ -311,10 +324,48 @@ void LinkedShader::UpdateUniforms(u32 vertType, const ShaderID &vsid, bool useBu
 		render_->SetUniformUI1(&u_depal_mask_shift_off_fmt, val);
 	}
 
+#ifdef OPENXR
```

Comment:
have a feeling that we're gonna end up with a lot of game profiles, setting up checks for what might be HUD or not...

Would be nice if we could find a way to centralize it in GPUStateUtils.cpp, as we will want to also support VR with Vulkan.

Reply:
I am trying to avoid per games configs as much as I can. I studied the PCVR version and I can say that 50% of the configs will not be necessary because I can detect it automatically. I hope I will find something similar for HUD.